### PR TITLE
ExplicitModule: remove all search paths for command lines to explicitly build Swift textual interface

### PR DESCRIPTION
**Pull request by nkcsgexi**

### CODE REVIEW

**Comment by nkcsgexi** [comment]
@swift-ci please smoke test

**Review by artemcm — APPROVED** [review]
Cool!

File: test/ScanDependencies/module_deps.swift [hunk 1]
```diff
@@ -133,7 +133,6 @@ import SubE
 // CHECK: "commandLine": [
 // CHECK: "-compile-module-from-interface"
 // CHECK: "-target"
-// CHECK: "-sdk"
```
Comment (DougGregor) [inline test/ScanDependencies/module_deps.swift:136]: Should we add some `CHECK-NOT`s here to make sure there aren't any of the `-I`'s here?

**Review by DougGregor — APPROVED** [review]
Cool!

File: test/ScanDependencies/module_deps.swift (hunk 1 above)
Comment (nkcsgexi) [reply test/ScanDependencies/module_deps.swift:136]: Yeah, will add.

**Comment by nkcsgexi** [comment]
@swift-ci please smoke test

**Comment by nkcsgexi** [comment]
@swift-ci Please smoke test OS X platform

**Comment by nkcsgexi** [comment]
@swift-ci Please smoke test OS X platform

**Pull request merged**